ui-meta/sample-appliance.yml

```diff
@@ -6,51 +7,52 @@ label: "Sample Appliance"
 
 # A list of parameters to be gathered from the user
 parameters:
-  - # The name of the Ansible variable that will be populated with the parameter value.
-    # This field is required.
-    name: load_balancer_flavor
+  # The name of the Ansible variable that will be populated with the parameter value.
+  # This field is required.
+  - name: load_balancer_flavor
     # A human-readable name for the parameter. Defaults to the name if not given.
     label: Load balancer size
-    # An optional short description for the parameter.
+    # An optional short description for the parameter.
     description: The size to use for the load balancer node.
-    # The kind of the parameter. This field is required.
+    # The kind of the parameter. This field is required.
     # Valid values are:
-    #   list
+    #   list
     #     A list of items.
     #   string
-    #     A string or something that can be coerced.
-    #   integer
-    #     An integer or a string that can be coerced.
-    #   number
-    #     Any number, i.e. an integer or a float, or a string that can be coerced.
-    #   boolean
+    #     A string or something that can be coerced.
+    #   integer
+    #     An integer or a string that can be coerced.
+    #   number
+    #     Any number, i.e. an integer or a float, or a string that can be coerced.
+    #   boolean
     #     A boolean flag.
-    #     Valid values are: true/false, 1/0, "1"/"0", "true"/"false" and "yes"/"no".
-    #   choice
+    #     Valid values are: true/false, 1/0, "1"/"0", "true"/"false" and "yes"/"no".
+    #   choice
     #     A value from a set of choices.
-    #   cloud.size
+    #   cloud.size
     #     The ID of a size in the target tenancy.
-    #   cloud.machine
-    #     The ID of a machine in the target tenancy.
+    #   cloud.machine
+    #     The ID of a machine in the target tenancy.
     #   cloud.ip
-    #     The ID of an external IP in the target tenancy.
+    #     The ID of an external IP in the target tenancy.
     #   cloud.volume_size
     #     An integer that represents the size of a volume that will be created.
     #     Used for quota calcuations.
-    #   cloud.volume
-    #     The ID of a volume in the target tenancy.
-    #   cloud.cluster
-    #     The ID of another cluster in the target tenancy.
+    #   cloud.volume
+    #     The ID of a volume in the target tenancy.
+    #   cloud.cluster
+    #     The ID of another cluster in the target tenancy.
     kind: "cloud.size"
     # Indicates whether the parameter is required. Defaults to true if not given.
     required: true
     # The default value value for the parameter. Defaults to null if not given, indicating no default.
-    default: null
+    default:
```

Comment:
Possibly this could change behaviour so we'll need to validate and add an exception if it does.